packages/components/pie-webc-core/src/internals/PieElement.ts

```diff
@@ -0,0 +1,7 @@
+import { LitElement } from 'lit';
```

Comment:
Just checking, Lit is still externalised even though it's imported inside `internals`?

Reply:
From looking in the built file for `pie-button`, I believe so. This is backed up by the fact that we still externalise Lit in the vite config for pie-webc-core too


<img width="1253" alt="Screenshot 2025-03-27 at 12 17 08" src="https://github.com/user-attachments/assets/d076692a-766e-435f-8e4d-96d6514c473e" />
<img width="611" alt="Screenshot 2025-03-27 at 12 18 24" src="https://github.com/user-attachments/assets/4f0cd61a-03c9-4ac1-8b64-050ab08a692e" />